internal/api/handlers/schedulers_handler.go

```diff
@@ -162,24 +159,24 @@ func (h *SchedulersHandler) UpdateScheduler(ctx context.Context, request *api.Up
 	}, nil
 }
 
-func (h *SchedulersHandler) fromApiCreateSchedulerRequestToEntity(request *api.CreateSchedulerRequest) *entities.Scheduler {
-	return &entities.Scheduler{
-		Name:     request.GetName(),
-		Game:     request.GetGame(),
-		State:    entities.StateCreating,
-		MaxSurge: request.GetMaxSurge(),
-		PortRange: &entities.PortRange{
-			Start: request.GetPortRange().GetStart(),
-			End:   request.GetPortRange().GetEnd(),
-		},
-		Spec: game_room.Spec{
-			Version:                request.GetVersion(),
-			TerminationGracePeriod: time.Duration(request.GetTerminationGracePeriod()),
-			Affinity:               request.GetAffinity(),
-			Toleration:             request.GetToleration(),
-			Containers:             h.fromApiContainers(request.GetContainers()),
-		},
-	}
+func (h *SchedulersHandler) fromApiCreateSchedulerRequestToEntity(request *api.CreateSchedulerRequest) (*entities.Scheduler, error) {
+	scheduler, err := entities.NewScheduler(
```

Comment:
Maybe we can return result of this method directly? we are not using `scheduler` or `err` variables, right?